### PR TITLE
Add recent changes to CHANGELOG.md for 2.3.4

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 * Shade: Fix library sidebar splitter glitch [#4828](https://github.com/mixxxdj/mixxx/pull/4828) [lp:1979823](https://bugs.launchpad.net/mixxx/+bug/1979823)
 * MacOS builds: Perform ad-hoc signing of macOS bundle in Pull request and personal repositories [#4774](https://github.com/mixxxdj/mixxx/pull/4774)
 * Waveform: Avoid visual glitch with ranges < 1 px [#4804](https://github.com/mixxxdj/mixxx/pull/4804)
+* Traktor S3: Fix issues with sampler and hotcue buttons [#4676](https://github.com/mixxxdj/mixxx/pull/4676)
+* Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15 [#4676](https://github.com/mixxxdj/mixxx/pull/[#4863](https://github.com/mixxxdj/mixxx/pull/4863))
```
Comment (ronso0) [inline CHANGELOG.md:12]:
```suggestion
* Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863))
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
just a copy/paste fail, otherwise LGTM

**Comment by daschuer** [comment]
Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 * Shade: Fix library sidebar splitter glitch [#4828](https://github.com/mixxxdj/mixxx/pull/4828) [lp:1979823](https://bugs.launchpad.net/mixxx/+bug/1979823)
 * MacOS builds: Perform ad-hoc signing of macOS bundle in Pull request and personal repositories [#4774](https://github.com/mixxxdj/mixxx/pull/4774)
 * Waveform: Avoid visual glitch with ranges < 1 px [#4804](https://github.com/mixxxdj/mixxx/pull/4804)
+* Traktor S3: Fix issues with sampler and hotcue buttons [#4676](https://github.com/mixxxdj/mixxx/pull/4676)
+* Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863))
```
Comment (Holzhaus) [inline CHANGELOG.md:12]:
```suggestion
* Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863)
```

**Comment by daschuer** [comment]
Upsi, Done

File: CHANGELOG.md (hunk 2 above)
Comment (ronso0) [reply CHANGELOG.md]: ups, slipped through. hard to notice with the regular diff, just figured there's a 'rich diff'

**Pull request merged**